Form/Type/SearchType.php

```diff
@@ -17,7 +19,7 @@ public function buildForm(FormBuilderInterface $builder, array $options)
         $builder
             ->add(
                 'term',
-                'text',
+                TextType::class,
```

Comment:
werkt 'text' niet meer? Want ik vind dat eerlijk gezegd wel wat properder dan ::class stuff (als ge niet moet de fully qualified name meegeven natuurlijk)

`::class` werkt ook enkel in PHP 5.5 en hoger.